tests/http/test_proxy_protocol.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+"""
+    proxy.py
+    ~~~~~~~~
+    ⚡⚡⚡ Fast, Lightweight, Pluggable, TLS interception capable proxy server focused on
+    Network monitoring, controls & Application development, testing, debugging.
+
+    :copyright: (c) 2013-present by Abhinav Singh and contributors.
+    :license: BSD, see LICENSE for more details.
+"""
+import unittest
+
+from proxy.http.parser import ProxyProtocol, PROXY_PROTOCOL_V2_SIGNATURE
+
+
+class TestProxyProtocol(unittest.TestCase):
```

Comment:
Did you know that inheriting from `unittest.TestCase` prevents you from using many powerful pytest features?

Reply:
Totally, lot of legacy finger memory :).  Lots of enhancements that we must make in these respect.  E.g. total `pytest` adoption.  May be upgrade linting from `autopep8` to `black`.  Hopefully some day soon :D

Reply:
Personally, I can't stand black because it doesn't let you make important decisions. But it's fine for newbies who don't know how to write code consistently. I find most of the autoformatters harmful in many because in some cases they silently hide legitimate problems that a machine cannot solve because of the lack of human context. This is why I opt-in for very specific formatting cases that are unlikely to shadow real problems.